src/averageAges.js

```diff
@@ -55,7 +59,40 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const mothers = getMothers(people);
+  const childs = people
+    .filter(v => v.mother)
```

Comment:
```suggestion
  const childs = people
    .filter(person => person.mother)
```
Давай будемо скрізь використовувати більш зрозумілі назви для змінних